test/unit/instrumentation/http/http.test.js

```diff
@@ -292,6 +283,8 @@ test('built-in http module instrumentation', (t) => {
           const segment = transaction.baseSegment
           const spanAttributes = segment.attributes.get(DESTINATIONS.SPAN_EVENT)
 
+          // TODO: these attrs aren't getting filtered
+          // not familiar with code so trying to trace where this is supposed to be removed
```

Comment:
I believe the `x-filtered-out` header is getting filtered out through attribute filtering wildcard rules. Looks like the regex test is `/^(?:response\.(?:headers\.(?:setCookie|x))|request\.(?:headers\.(?:setCookie|x)))/g` for filtering out.

Reply:
yea i forgot to remove TODO after i traced that